tests/Makefile

```diff
@@ -328,12 +328,14 @@ execute: $(basename $(TARGET))
 	$(CL_CXX) -cc1 -emit-llvm-bc -triple spir64-unknown-unknown \
 	  -include $(triSYCL_DIR)/include/CL/sycl/detail/SPIR/opencl_spir.h -o $@ $<
 
+# temporarily removed -reqd-workgroup-size-1 from below kernel compilation step until 
+# the changes are merged in
 # Process bitcode with SYCL passes to generate kernels
 %.kernel.bc: %.pre_kernel.ll
 	# To debug, think about using options like -debug -stats -print-after-all...
 	$(LLVM_BUILD_DIR)/bin/opt -load $(LLVM_BUILD_DIR)/lib/SYCL.so \
 	  -globalopt -deadargelim -SYCL-args-flattening -deadargelim \
-	  -SYCL-kernel-filter -globaldce -RELGCD -reqd-workgroup-size-1 -inSPIRation \
+	  -SYCL-kernel-filter -RELGCD -inSPIRation -globaldce \
```

Comment:
I would keep the `-globaldce` pass before the `-RELGCD` to remove the host part because I am unsure that `-inSPIRation` can work with a messy code coming from complex programs otherwise.
So just add the `-globaldce` pass after and do not change anything else for now.